kalite/control_panel/templates/control_panel/partials/_facility_table.html

```diff
@@ -27,7 +39,7 @@
                     </thead>
                     <tbody>
                         {% for id,facility in facilities.items %}
-                            <tr>
+                            <tr {% if facility.meta_data_in_need %} class="warning" {% endif %}>
```

Comment:
I like that idea. Currently you have to notice the edit icon on the subsequent facility page to actually make a change. Will fix this and the other issues. Thanks for helping so much with this first PR. Hopefully future ones will be a bit cleaner!